docs/VictoriaLogs/backup.md

```diff
@@ -0,0 +1,62 @@
+---
+weight: 6
+title: Backup and Restore
+menu:
```

Comment:
I'd suggest to have this at /victorialogs/ root doc due to the following reasons:
1. I'm not sure if backup/restore should be a separate menu option, because in VM it is not like this
2. Once added, it will be extremely painful to remove this option from main menu in future.

Reply:
Moved to `## Backup and restore` under `https://docs.victoriametrics.com/victorialogs/`.